packages/react-ui-lib/src/binding/persist.tsx

```diff
@@ -68,3 +67,28 @@ export const PersistOnFieldChange = ({ field }: {
 
 	return null
 }
+
+export const PersistOnRelationChange = ({ field }: { field: string }) => {
```

Comment:
We should either support sugared nested paths, or validate that it is single field (i.e. It does not contain a dot)

Reply:
We should definitely add support for sugared nested paths. I can see the confusion, if only single field works.